tests/console/python3_new_version_check.pm

```diff
@@ -32,16 +32,19 @@ sub run {
         add_suseconnect_product('sle-module-python3');
     }
 
-    my @python310_results = split "\n", script_output("zypper se python310 | awk -F \'|\' \'/python310/ {gsub(\" \", \"\"); print \$2}\'");
-    if ((scalar(@python310_results) == 0) || ($python310_results[0] ne "python310")) {
-        die("Python 3.10 not found in the repositories\n\n");
+    # Test newer python311 in newer release
+    my $subversion = is_sle('<=15-SP4') ? 10 : 11;
```

Comment:
The main use case is for Tumbleweed, right?

Since there are multiple python versions available on TW, I would recommend to test all of them. We could introduce a variable e.g. `PYTHON_VERSIONS` and check all versions defined there.

This means we can move the python version from hardcoded values in the test code to test settings, and we can have multiple versions checked at the same time.

WDYT?

Reply:
That would be great, let me do